README.md

```diff
@@ -103,11 +103,11 @@ vagrant@ceph-admin:~/test-cluster$ ceph-deploy mon create-initial
 And our two OSDs. For these, we need to log into the server machines directly:
 
 ```console
-vagrant@ceph-admin:~/test-cluster$ ssh ceph-server-2 sudo mkdir /var/local/osd0
+vagrant@ceph-admin:~/test-cluster$ ssh ceph-server-2 sudo "sudo mkdir /var/local/osd0 && sudo chown ceph:ceph /var/local/osd0"
```

Comment:
Running a test local when I run the following command:
sudo mkdir /tmp/abc && chown brownie:brownie /tmp/abc

I get a permission denied error. If I running the command with a sudo on the front of the chown then it works. The && is like running two separate commands so you need the extra sudo.